api/lib/schema.js

```diff
@@ -267,6 +267,8 @@ const typeDefs = gql`
     proposer: String
     initialDeposit: InputCoin
     voteOption: String
+    lockedBalance: Float
+    conviction: Int
```

Comment:
can we call this something understandable?

Reply:
how would you call conviction? `timeLock`, right?

Reply:
I renamed it to `timeLock`